appveyor.yml

```diff
@@ -6,6 +6,20 @@ cache:
   - c:\cargo\git
   - c:\projects\habitat\target
 
+skip_commits:
+  # version bumps by Expeditor happen as a separate commit after the merge, we can skip
+  author: Chef Expeditor
```

Comment:
Unfortunately, releases are currently triggered on commits from Expeditor.

We tried this earlier, but it ended up killing our ability to do a release 😱 Guess we didn't get the proper fix in place :frown:

On the upside, I think we've only got one more release to do from Appveyor before we've got everything moved over to Buildkite.